docs/projectSetup/iOS/checklist_iOS.md

```diff
@@ -0,0 +1,58 @@
+# iOS Project Setup
+## Git
+Create new repository based on native-ios-repo-template
+
+## Apple Developer
+The Developer Certiciate and Provisioning Profile are required for code signing and distribution.  
+
+[Apple Developer Portal](https://developer.apple.com/account/resources/)
+
+1. Load the NanoGiants GmbH Developer Certificate, and add it into your Keychain.
+2. add identifiers for the different environments.
+3. add provisioning profiles for the different environments.
+
+## Xcode
+### Create an empty base project in Xcode.
+Configure signing & capabilities for different environments.
+
+1. Debug
+2. Development
+3. Staging
+4. Production
+   
+### Configure analytics, crash reporting, and static file analysis tools.
+
+1. Integrate Sentry for error tracking.
+2. Use SonarCloud for code quality analysis.
+3. Configure Android Lint for static code analysis.
+4. Set up Detekt for Kotlin code analysis.
```

Comment:
Is this correct? Seems to be from the Android Checklist.

Reply:
Same here.